lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -121,17 +121,27 @@ <h2 class="hd hd-2 instructor-dashboard-title">${_("Instructor Dashboard")}</h2>
         ## when the javascript loads, it clicks on the first section
         <ul class="instructor-nav">
           % for section_data in sections:
+            <%
+              is_hidden = False
```

Comment:
Can this be rewritten with getattr (which allows you to specify the default value of False)?